WcaOnRails/app/models/competition.rb

```diff
@@ -660,6 +660,14 @@ def trainee_delegate_ids
     @trainee_delegate_ids || trainee_delegates.map(&:id).join(",")
   end
 
+  def enable_v2_registrations
```

Comment:
By Ruby convention, methods that modify an existing object in-place should end with an exclamation mark

Reply:
I thought ! means it returns an error

Reply:
like find! vs. find